routes/index/sold-tracking.html

```diff
@@ -0,0 +1,8 @@
+<!-- Twitter single-event website tag code -->
+<!--
+<script src="//platform.twitter.com/oct.js" type="text/javascript"></script>
+<script type="text/javascript">twttr.conversion.trackPid('nxrhi', { tw_sale_amount: 0, tw_order_quantity: 0 });</script>
+-->
+<img height="1" width="1" style="display:none;" alt="" src="https://analytics.twitter.com/i/adsct?txn_id=nxrhi&p_id=Twitter&tw_sale_amount=0&tw_order_quantity=0" />
+<img height="1" width="1" style="display:none;" alt="" src="//t.co/i/adsct?txn_id=nxrhi&p_id=Twitter&tw_sale_amount=0&tw_order_quantity=0" />
```

Comment:
yeah, it's weird, but it does. and of course ideally it'd want to run the scripts, but I don't want to deal with the javascript. 😛